apps/panel/src/pages/landing/Landing.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import {BaseInput, Button, FlexElement, Icon, Input, StringInput, Text} from "oziko-ui-kit";
+import styles from "./Landing.module.scss";
+import logoSmall from "../../assets/images/logo_small.svg";
+import logo from "../../assets/images/logo.svg";
+import "oziko-ui-kit/dist/index.css";
+
+const Landing = () => {
+  return (
+    <div className={styles.container}>
+      <FlexElement direction="vertical" gap={5}>
+        <FlexElement
+          className={styles.contentContainer}
+          dimensionX={400}
+          direction="vertical"
+          gap={10}
+        >
+          <img src={logo} alt="logo"></img>
+          <StringInput label="Name"></StringInput>
+          <BaseInput
+            dimensionX={"fill"}
+            labelProps={{
+              dimensionX: "hug",
+              divider: true,
+              prefix: {
+                children: <Icon className={styles.passwordIcon} name="eye" />
+              },
+              root: {
+                dimensionX: "hug",
+                children: (
+                  <Text className={styles.passwordText} size="medium">
+                    Password
+                  </Text>
+                )
+              }
+            }}
+          >
+            <Input value={""} onChange={() => {}} />
+          </BaseInput>
+          <Button fullWidth>
+            <Icon name="login"></Icon>
```

Comment:
use the self-closing format
```suggestion
            <Icon name="login" />
```

Reply:
Done